agent-engines/gpu_worker/decentralized_orchestrator.py

```
@@ -0,0 +1,123 @@
+
+    def get_cluster_state(self) -> List[NodeInfo]:
+        """Return the current state of all nodes in the cluster."""
+        local_load = sum(w.load for w in self.pool._workers) / len(self.pool._workers) if self.pool._workers else 0.0
+        local_node = NodeInfo(
+            node_id=self.node_id,
+            address="localhost",
+            status="online",
+            load=local_load,
+            last_seen=datetime.now(timezone.utc)
+        )
+        return [local_node] + list(self.peers.values())
```

Ruff (0.15.11)

[warning] Consider `[local_node, *list(self.peers.values())]` instead of concatenation

Replace with `[local_node, *list(self.peers.values())]`

(RUF005)